website/sidebars.js

```diff
@@ -117,8 +117,15 @@ module.exports = {
     "dbt-cli/configure-your-profile",
   ],
   "dbt Cloud": [
-    "docs/dbt-cloud/cloud-overview",
-    "docs/dbt-cloud/cloud-quickstart",
+    {
+      type: "category",
+      label: "Introduction",
+      link: { type: 'doc', id: 'docs/dbt-cloud/cloud-intro' },
```

Comment:
@JKarlavige Is it possible to upgrade our docusaurus to 2.0.0-beta.14? I want to use this link ability but can't currently use it (the build fails). See the [category doc link ](https://docusaurus.io/docs/sidebar#category-doc-link) docs for more context.

Thank you 🙏🏻